plugins/pv_lxc.c

```diff
@@ -191,22 +191,44 @@ static void pv_setup_lxc_container(struct lxc_container *c,
 		c->set_config_item(c, "lxc.log.level", log_level);
 	}
 	if (p->mgmt) {
-		c->set_config_item(c, "lxc.mount.entry", "/pv pantavisor"
-							" none bind,ro,create=dir 0 0");
-		c->set_config_item(c, "lxc.mount.entry", "/pv/logs pantavisor/logs"
-							" none bind,ro,create=dir 0 0");
-		c->set_config_item(c, "lxc.mount.entry", "/pv/user-meta pantavisor/user-meta"
-							" none bind,ro,create=dir 0 0");
+		snprintf(entry, sizeof (entry),
```

Comment:
why not SNPRINTF_WTRUNC here?

Reply:
Let me try out SNPRINTF_WTRUNC again, there might be a problem with logs not being initialized here, so printing the warning might fail. I did the changes some time ago, so I am not sure if that was the case. I will report back soon.